resources/fetchers/monitoring_fetcher.go

```diff
@@ -109,17 +109,9 @@ func (s SecurityHubResource) GetData() any {
 }
 
 func (s SecurityHubResource) GetMetadata() (fetching.ResourceMetadata, error) {
-	if s.DescribeHubOutput == nil || s.HubArn == nil {
-		return fetching.ResourceMetadata{
-			ID:      "",
-			Name:    "",
-			Type:    fetching.MonitoringIdentity,
-			SubType: fetching.SecurityHubType,
-		}, nil
-	}
 	return fetching.ResourceMetadata{
-		ID:      *s.HubArn,
-		Name:    *s.HubArn,
+		ID:      s.GetResourceArn(),
```

Comment:
There will be a collision when there are multiple `account_ids` that are sending the evaluation for each of their regions.
I think that maybe the `account_id` should be part of the ARN (when the hub is disabled) in order to avoid collisions.

Reply:
done